pywbem_mock/_wbemconnection_mock.py

```diff
@@ -449,7 +450,117 @@ def add_cimobjects(self, objects, namespace=None):
                 except KeyError:
                     self.qualifiers[namespace] = NocaseDict({qual.name: qual})
             else:
-                assert False, 'add_cimobjects. %s not valid.' % type(obj)
+                assert False, 'Object to add_cimobjects. %s invalid type' \
+                              % type(obj)
+
+    def subscribe_method(self, classname, methodname, method_callback,
+                         namespace=None,):
+        """
+        Add a callback for a method that will be called by the client when
+        InvokeMethod is executed.  This method does no real validation of
+        the classname, methodname or method_callback in put parameters.
+
+          Parameters:
+            classname (:term:`string`):
+                The classname for the method defined by methodname.
```

Comment:
Clarify in the description whether classname must be:
(1) The CIM class that defines the method.
(2) The creation class of CIM instances or the CIM class against which the method will be invoked.

Reply:
Good catch, In fact, I defined to be exactly that the class defined for the method and do not allow for subclassing for the method call. That will also have to change.